ui/browser/views/navbar/navbar.jsx

```diff
@@ -20,8 +20,8 @@ import Location from './location.jsx';
 function NavBar(props) {
   const {
     navBack, navForward, navRefresh, page, pages, openMenu, ipcRenderer,
```

Comment:
>  this list of props seems untenable over time

agreed, if you look at diff there's copy/paste required across lots of files to add an action like this.  We should discuss further to see if this is how we want to continue as more actions get added